test/reload_system.bats

```diff
@@ -6,11 +6,23 @@ load helpers
 function setup() {
 	setup_test
 	TEST_IMAGE=quay.io/saschagrunert/hello-world
+	TEST_IMAGE2=quay.io/crio/hello-wasm
 	export CONTAINER_REGISTRIES_CONF="$TESTDIR/containers/registries.conf"
-	mkdir "$TESTDIR/containers"
+	export CONTAINER_REGISTRIES_CONF_DIR="$TESTDIR/containers/registries.conf.d"
+	mkdir -p "$CONTAINER_REGISTRIES_CONF_DIR"
 	printf "[[registry]]\nlocation = 'quay.io/saschagrunert'\nblocked = true" \
 		>> "$CONTAINER_REGISTRIES_CONF"
-	start_crio
+	printf "[[registry]]\nlocation = 'quay.io/crio'\nblocked = true" \
+		>> "$CONTAINER_REGISTRIES_CONF_DIR/01-registry.conf"
+}
+
+function drop_in_for_automatically_reload_registries() {
+	cat << EOF > "$CRIO_CONFIG_DIR/00-automatically-reload-registries.conf"
```

Comment:
```suggestion
	cat <<EOF > "$CRIO_CONFIG_DIR/00-automatically-reload-registries.conf"
```

Reply:
It looks pretty consistent with the other tests, unless my editor is a bit off

Reply:
Oh, if so, then ignore me. It was a habit while reviewing the code.